src/util/util.ts

```diff
@@ -110,7 +110,7 @@ export class StrUtil {
    * @param indicator - The indicator to use when the string is too long.
    */
   public static naturalLimit(str: string, limit: number, indicator?: string): string {
-    const newIndicator = indicator == null ? '...' : indicator;
+    const newIndicator = indicator ?? '...';
```

Comment:
this can be a default value in the parameter right?

Reply:
Fixed